connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -298,10 +295,15 @@ object KustoWriter {
           Try(
             ingestClient.ingestFromBlob(new BlobSourceInfo(blobUri + sas, size, UUID.randomUUID()), props)
           ) match {
-            case Success(x) => x
+            case Success(x) =>
+              val containerWithSas = new ContainerWithSas(blobResource.blob.getStorageUri.getPrimaryUri.toString+blobResource.sas, null)
```

Comment:
So this PR also includes the ingestion report ...? Please add to PR description

Reply:
Updated the title